index.src.html

```diff
@@ -615,7 +622,15 @@ <h4 id="secure-context-restriction">Secure context restriction</h4>
 
           1.  If |connection|'s [=connection/IP address space=] is
               [=IP address space/less public=] than |clientAddressSpace|, then
-              return a [=network error=].
+              show a [=permission prompt=].
+
+          1. If |permission|'s result is:
```

Comment:
It seems that the "permission" variable is not defined before it's used.
Also I feel like "permission's result" is a little awkward.
Maybe something like below:

```suggestion
  show a <a data-link-type="dfn" href="#private-network-access-permission-prompt" id="ref-for-private-network-access-permission-prompt②">permission prompt</a> and ask for the user's <dfn ...>permission</dfn>.</p>
         <li data-md>
          <p>If <var>permission</var> is:</p>
```